### PR TITLE
fix: revert breaking changes

### PR DESCRIPTION
**Pull request by n1ru4l**
This reverts the breaking changes I did in https://github.com/relay-tools/relay-compiler-language-typescript/pull/267 which broke Typescript < 4 support in 13.0.3 in a patch release :( (https://github.com/relay-tools/relay-compiler-language-typescript/issues/272).

We wanna release this as 13.0.5 then do a major release (14.0.0), which drops typescript < 4 support.

@alloy It seems like Travis CI automatically creates a release on every push on `master`. Could you explain how I could achieve a major version release? Is it based on commit messages? Maybe we should add a small contribution guide that explains how the deployment/changelog/versioning is done?

**Edit:** I just figured out this repo is using `auto`. Can you confirm that assigning the labels will trigger the right version bump being made? https://intuit.github.io/auto/docs/welcome/getting-started

### CODE REVIEW

File: src/TypeScriptGenerator.ts [hunk 1]
```diff
@@ -51,14 +51,14 @@ export const generate: TypeGenerator["generate"] = (schema, node, options) => {
   const printer = ts.createPrinter({ newLine: ts.NewLineKind.LineFeed });
 
   const resultFile = ts.createSourceFile(
-    "graphql-def.ts",
+    "grapghql-def.ts",
```
Comment (sibelius) [inline src/TypeScriptGenerator.ts:54]: Os this a typo?

**Review by sibelius — APPROVED** [review]

File: src/TypeScriptGenerator.ts (hunk 1 above)
Comment (n1ru4l) [reply src/TypeScriptGenerator.ts:54]: IT was a typo previously, I just reverted the whole commit.

Comment (n1ru4l) [reply src/TypeScriptGenerator.ts:54]:
```suggestion
    "graphql-def.ts",
```

**Comment by alloy** [comment]
:rocket: PR was released in v13.0.5 :rocket:

**Comment by alloy** [comment]
> I just figured out this repo is using auto. Can you confirm that assigning the labels will trigger the right version bump being made?

@n1ru4l That is correct 👍

**Pull request merged**